sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -14,43 +14,50 @@
     </item>
     <item>
       <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <Description>Time at which the most recent data was read (TAI unix seconds).</Description>
       <IDL_Type>double</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>ux</EFDB_Name>
-      <Description>x-axis wind speed.</Description>
+      <EFDB_Name>speed</EFDB_Name>
+      <Description>Median wind speed (x, y, z) measured over a period on the order of a minute.</Description>
```

Comment:
I'd prefer that the period over which the speed is reported is determined by the configuration of the CSC. If you agree then please amend this Description and other similar ones below accordingly.

Reply:
The period will be a combination of the sensor configuration (data reporting interval) and CSC configuration (how many samples to aggregate). I'm not sure how to say that concisely in the XML. Any suggestions?